ambari-server/src/main/resources/Ambari-DDL-Oracle-CREATE.sql

```diff
@@ -1039,6 +1039,29 @@ CREATE TABLE upgrade_history(
   CONSTRAINT UQ_upgrade_hist_srvc_grp UNIQUE (upgrade_id, service_group_id)
 );
 
+CREATE TABLE upgrade_plan (
+  id NUMBER(19) NOT NULL,
+  cluster_id NUMBER(19) NOT NULL,
+  upgrade_type VARCHAR2(255) DEFAULT 'ROLLING' NOT NULL,
+  direction VARCHAR2(255) DEFAULT 'UPGRADE' NOT NULL,
+  skip_failures SMALLINT DEFAULT 0 NOT NULL, 
+  skip_sc_failures SMALLINT DEFAULT 0 NOT NULL, 
+  skip_prechecks SMALLINT DEFAULT 0 NOT NULL,
```

Comment:
Do we use `SMALLINT` for Oracle or `NUMBER(5)`?

Reply:
The file has a mix of NUMBER(1), SMALLINT.  i only see one NUMBER(5) and it appears to be like a short (it's a count).

Reply:
OK - if we've previously had `SMALLINT` then I'm OK with it. `NUMBER(1)` is what I'd expect for booleans in this file.